eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -41,8 +41,8 @@ steps:
         displayName: Install devcerts
 
     - ${{ if eq(parameters.isWindows, 'true') }}:
-      - script: dotnet dev-certs https --check --verbose
-        displayName: Check dev-certs are installed
+      - script: dotnet dev-certs https --trust
```

Comment:
Do we need the cert trusted or just installed?

Reply:
Do our tests need that to be trusted? I'll try without the `--trust` and see if anything fails.

Reply:
Nothing failed, but the machine that it ran on already had a certificate installed. I'm going to merge this right now to unblock the internal build. And we can watch if anything fails on `main` and needs `--trust`.

Reply:
https://github.com/dotnet/aspire/pull/4742#pullrequestreview-2152395467